src/lib/validations/ExtendedValidatorBase.ts

```diff
@@ -27,10 +28,11 @@ export class ExtendedValidatorBase {
    * - jsonSchema based
    * - programmatic logic
    */
-  static runAllValidations(block: IBlock, schemaVersion: string, options?: unknown): IValidationStatus {
+  static runAllValidations(block: IBlock, schemaVersion: string, customSchema?: JSONSchema7, options?: unknown): IValidationStatus {
     const validationStatus = validateBlockWithJsonSchema({
       block,
       schemaVersion,
+      customBlockJsonSchema: customSchema,
```

Comment:
This PR is really all about allowing us to pass `customBlockJsonSchema` to `runAllValidations` method. It is supposed to be used in Voto5 to supply the schema from `flow-block-runners`